src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav_bar {
+  margin-right: 50px;
+}
+
+.logo {
+  text-decoration: none;
+  line-height: 0;
+  margin-left: 50px;
```

Comment:
Get rid of margins, specify these spacing with `padding: 0 50px;` to the header element